pkg/reader/reader.go

```diff
@@ -150,3 +154,28 @@ func (r *Reader) detectFormat(rs io.ReadSeeker) (formats.Format, error) {
 	}
 	return format, nil
 }
+
+// Retrieve reads a document from the configured storage backend using the
+// default options.
+func (r *Reader) Retrieve(id string, o *Options) (*sbom.Document, error) {
+	return r.RetrieveWithOptions(id, defaultOptions)
```

Comment:
should this use `o`, and fall back to `defaultOptions` if `o` is nil?

Reply:
Or, I suppose after reading the comment, should `o` be dropped as a parameter

Reply:
Oooh nice catch, thank you @mikhailswift!!! :